documentation/tutorials/getting-started-with-ash-and-phoenix.md

```diff
@@ -53,62 +53,12 @@ $ cd my_ash_phoenix_app
 
 ### Add Dependencies
```

Comment:
I think we can take this whole Add Dependencies section out :) `mix deps.get` shouldn't be required after igniter installation.

Reply:
That's fine. I wasn't sure if `mix.deps.get` was still needed.